backend/config/routes.rb

```diff
@@ -10,6 +10,7 @@
       post '/sign-out', to: 'sessions#sign_out'
       resource :users, only: [:update] do
         put 'me', to: 'users#update'
+        get 'me', to: 'users#show'
```

Comment:
Could you create a follow-up task to split this logic into ProfilesController and UsersController later? Thanks!